The-Chonky-Panda/Atelier
# More reviews functionality

## Description

'More Reviews` button should load more reviews on the page.

## How to test

`npm test` should still pass
Manual test: 
1. click on the `More reviews` button, verify that 2 more reviews load to the list
2. if there are still unloaded reviews, the button would remain there, repeat and verify step 1
3. if there are no more reviews, the button should disappear

## Notes


## Issue tracking

https://trello.com/c/34DcYst5
